src/cfn_secrets_manager_secret_provider.py

```diff
@@ -95,15 +101,26 @@ def create_arguments(self):
         return args
 
     def set_return_attributes(self, response):
-        self.set_attribute('VersionId', response['VersionId'])
```

Comment:
This value was not in the response when I created a secret, so I had to remove it. Not sure what the intention of this line is, so might need to discuss adding it back....